### PR TITLE
Added Navbar, Logo and Actions (icons) in header.liquid

### PR DESCRIPTION
**Pull request by anishdalvi**
Solved #39 

Crucial Change in `theme.liquid` : Added `dev_header` section to replace the previous `header` section
Also, added logo files in assets as _Asset should be served by the Shopify CDN for better performance._

Warnings:

image

<hr>

Demo Video : [View](https://drive.google.com/file/d/1hH__0M1YpYbIQ6KYD5tQi6Jx10D_CfTC/view?usp=sharing)

_Note: The CSS can be improved further._

### CODE REVIEW

File: sections/dev_header.liquid [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{{ 'dev_header.css' | asset_url | stylesheet_tag }}
+
+<header class="header">
+    <button class="navbar-toggler" onclick="toggleNavbar()">
+        <span class="navbar-toggler-icon">&#9776;</span>
+    </button>
+    <div class="logo" id="logo">
+        <img
```
Comment (mihir-bombay-studio) [inline sections/dev_header.liquid:8]: We don't need the lazy loading tag here as header image will always be first one to be displayed, you can use the [image tag](https://shopify.dev/docs/api/liquid/filters/image_tag) here which now also automatically detects which section is at which position and automatically adds lazy loading

File: sections/dev_header.liquid [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            {%- endfor %}
+        </ul>
+    </nav>
+    <div class="actions">
+      {% if section.settings.search == true %}
+        <a href="#"><img src="{{ 'search-icon.png' | asset_url }}" alt="Search Icon" width=20 height=auto loading=lazy /></a>        
+      {% endif %}
+      {% if section.settings.cart == true %}
+        <a href="#"><img src="{{ 'cart-icon.png' | asset_url }}" alt="Cart Icon" width=20 height=auto loading=lazy /></a>
+      {% endif %}
```
Comment (mihir-bombay-studio) [inline sections/dev_header.liquid:49]: Use SVG images wherever possible especially for icons. Use them as snippets so we can also render them with certain classes to reuse and customize the look of the icons if required.

**Review by mihir-bombay-studio — CHANGES_REQUESTED** [review]
Nice addition! lets just optimise the images and it should be good for now

**Comment by anishdalvi** [comment]
Changes Done with additional feature of displaying alt text initially at header. 
**Used SVG for the icons** ✔️

**Comment by anishdalvi** [comment]
Changes Done.
Used SVG for the icons ✔️

Please review the updated branch 😃

**Pull request merged**